library/src/main/kotlin/wu/seal/jsontokotlin/JsonToKotlinBuilder.kt

```diff
@@ -0,0 +1,249 @@
+package wu.seal.jsontokotlin
+
+import extensions.chen.biao.KeepAnnotationSupport
+import extensions.jose.han.ParcelableAnnotationSupport
+import extensions.ted.zeng.PropertyAnnotationLineSupport
+import extensions.wu.seal.*
+import extensions.xu.rui.PrimitiveTypeNonNullableSupport
+import wu.seal.jsontokotlin.interceptor.InterceptorManager
+import wu.seal.jsontokotlin.test.TestConfig
+import wu.seal.jsontokotlin.utils.ClassImportDeclaration
+
+
+/**
+ * To get Kotlin class code from JSON
+ */
+class JsonToKotlinBuilder {
+
+    private var packageName = ""
+
+    init {
+        TestConfig.apply {
+
+            isTestModel = true
+            isCommentOff = true
+            isOrderByAlphabetical = true
+            isPropertiesVar = false
+            targetJsonConvertLib = TargetJsonConverter.None
+            propertyTypeStrategy = PropertyTypeStrategy.AutoDeterMineNullableOrNot
```

Comment:
this default should be `PropertyTypeStrategy.NotNull`

Reply:
Okay